### PR TITLE
Call Note List View

### PR DESCRIPTION
**Pull request by peterp**

### CODE REVIEW

**Comment by peterp** [comment]
@justinvdm ready for review

**Comment by peterp** [comment]
screen shot 2016-09-28 at 19 04 01

File: src/views/CallNoteList/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        uid="dismiss"
+        type={HeaderIcon.types.dismissDark}
+        onPress={onDismissPress}
+      />
+    </Header>
+    <ScrollView>
+      {callNotes.map((callNote, index) =>
+        <TouchableNativeFeedback
+          callNoteId={callNote.id}
+          key={index}
```
Comment (justinvdm) [inline src/views/CallNoteList/index.js]: Wondering if we should key by the call note id, in case the ordering of these changes for some reason.

**Comment by justinvdm** [comment]
Looks good :) One minor comment, otherwise 👍

**Pull request merged**